style.css

```diff
@@ -721,3 +720,192 @@ li {
     line-height: 32px;
   }
 }
+
+.card-button-mobile.button {
+  display: flex;
+  justify-content: space-between;
+  background-color: #36b37e;
+  border-radius: 7px;
+  color: white;
+  padding: 12px 12px;
+  text-align: center;
+  font-size: 17px;
+  border: none;
+  height: 48px;
+  gap: 12px;
+}
+
+@media (max-width: 1051px) {
+  .btn-close-details {
+    background: #ebecf0;
+    position: relative;
+    right: 0;
+    left: 0;
+    grid-column: -2 / -1;
+    grid-row: 1/2;
+  }
+}
+
+@media (min-width: 1050px) {
+  .tag-mobile {
+    background: #ebf0ee;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 0.75em;
+    line-height: 1em;
+    letter-spacing: 0.03em;
+    color: #344563;
+    border-radius: 0.5em;
+  }
+
+  .card-work-details-mobile {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 60px;
+    grid-template-rows: 60px repeat(5, auto);
+  }
```

Comment:
- [ ] As the previous Code Reviewer mentioned, you should apply some style to your mobile to make it look like the Figma template and fit the screen of the user. There are some differences of your style from the Figma template like the spacing between the buttons, the fonts inside the button, the colors of the button, the margin around the modal, etc. Also on different mobile screens like the Iphone SE, the modal does not fit inside the screen making it possible to scroll down, which is not a good user experience. Please fix the responsiveness of your modal to make it fit on every screen size and also to look more like the Figma template.
 - [ ] The same considerations above applies to the desktop view. It looks a little bit different from the Figma template. The design is not following the Figma template desktop design, it is following the same mobile design, and its larger than the viewport. Please try to make your modal look as much as possible as the Figma template.
 -Iphone SE view:
<img width="479" alt="image" src="https://user-images.githubusercontent.com/61919617/188918632-c9bd99fc-e681-41f8-a466-cff08c1732ad.png">
-Desktop view:
<img width="1146" alt="image" src="https://user-images.githubusercontent.com/61919617/188919285-b7163552-a4fc-4a32-812b-19eddb33c9b6.png">
-Figma template desktop view:
<img width="686" alt="image" src="https://user-images.githubusercontent.com/61919617/188919475-4e785a02-f7e7-44a2-89ff-892cfdf4e195.png">